simtools/applications/validate_file_using_schema.py

```diff
@@ -71,10 +79,21 @@ def _validate_yaml_or_json_file(args_dict, logger):
         logger.error(f"Input file {args_dict['file_name']} not found")
         raise
 
-    metadata_model.validate_schema(data, args_dict["schema"])
+    if args_dict.get("schema", None) is None and "meta_schema_url" in data:
+        args_dict["schema"] = data["meta_schema_url"]
+        logger.debug(f'Using schema from meta_schema_url: {args_dict["schema"]}')
```

Comment:
The quotes here (and below) should be reversed, `"` should be the outer ones and `'` should be the inner. 

(Unfortunately, despite me switching jobs, you still have to put up with such annoying comments sometimes...)

Reply:
Thanks - you keep being annoying :-)

(this code has been removed; so no issue anymore)